bundles/admin/admin-layereditor/view/LayerWizard.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Steps, Step } from '../components/Steps';
+import { LayerTypeSelection } from './LayerWizard/LayerTypeSelection';
+import { LayerURLForm } from './LayerWizard/LayerURLForm';
+import { LayerDetails } from './LayerWizard/LayerDetails';
+import { LayerCapabilitiesListing } from './LayerWizard/LayerCapabilitiesListing';
+import { Button } from '../components/Button';
+import { LayerWizardService } from './LayerWizard/LayerWizardService';
+
+export class LayerWizard extends React.Component {
+    constructor (props) {
+        super(props);
+        this.service = new LayerWizardService(() => this.setState({ layer: this.service.getLayer() }));
```

Comment:
I think we should reserve the name "service" for global singleton objects managing state and server communication. Here we have a "state manager" object that handles only state for this specific form instance. Instead we could implement the _container & presentational component_ pattern:
https://medium.com/@yassimortensen/container-vs-presentational-components-in-react-8eea956e1cea